Do not attempt to build documentation for db4 stuff,
we removed it in %prep

Lubomir Kundrak <lkundrak@redhat.com>

diff -urp lucene-2.3.0.orig/build.xml lucene-2.3.0/build.xml
--- lucene-2.3.0.orig/build.xml	2008-01-21 02:29:41.000000000 +0100
+++ lucene-2.3.0/build.xml	2008-02-19 16:52:14.000000000 +0100
@@ -237,8 +237,8 @@
           <packageset dir="contrib/analyzers/src/java"/>
           <packageset dir="contrib/ant/src/java"/>
           <packageset dir="contrib/benchmark/src/java"/>
-          <packageset dir="contrib/db/bdb-je/src/java"/>
-          <packageset dir="contrib/db/bdb/src/java"/>
+          <!-- <packageset dir="contrib/db/bdb-je/src/java"/> -->
+          <!-- <packageset dir="contrib/db/bdb/src/java"/> -->
           <packageset dir="contrib/highlighter/src/java"/>
           <packageset dir="contrib/lucli/src/java"/>
           <packageset dir="contrib/memory/src/java"/>
